armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -369,15 +367,35 @@ EOF
 
 
 # LIGHTNING --------------------------------------------------------------------
+BIN_DEPS_TAG="v0.0.1-alpha"
 LIGHTNING_VERSION="0.7.0"
 
-cd /usr/local/src/
-git clone https://github.com/ElementsProject/lightning.git || true
-cd lightning
-git checkout v${LIGHTNING_VERSION}
-./configure
-make
-make install
+## either compile c-lightning from source (default), or use prebuilt binary
+if [ "${BASE_BUILD_LIGHTNINGD}" == "true" ]; then
+  apt install -y  autoconf automake build-essential git libtool libgmp-dev \
+                  libsqlite3-dev python python3 net-tools zlib1g-dev libsodium-dev
+
+  cd /usr/local/src/
+  git clone https://github.com/ElementsProject/lightning.git || true
```

Comment:
I know it's not new in this change, but I'm curious if we still need the `|| true` here.. in which circumstances would a failure to clone the repo we're trying to build be okay, and we still would like to continue with the build?